### PR TITLE
mohammad/brand

### PR DESCRIPTION
**Pull request by binary-hamedanchi**
Add brand in connection URL

### CODE REVIEW

File: src/LiveApi.js [hunk 1]
```diff
@@ -81,7 +83,7 @@ export default class LiveApi {
     }
 
     connect(connection: ?WebSocket): void {
-        const urlPlusParams = `${this.apiUrl}?l=${this.language}&app_id=${this.appId}`;
+        const urlPlusParams = `${this.apiUrl}?l=${this.language}&app_id=${this.appId}${this.brand ? `&brand=${this.brand}` : ''}`;
```
Comment (borisyankov) [inline src/LiveApi.js]: I don't think we need the '? :' part?
What happens if we just pass '&brand=' ?

Comment (aminmarashi) [inline src/LiveApi.js]: We definitely should test that.

Comment (aminmarashi) [inline src/LiveApi.js]: I can confirm that `&brand=` works fine with current unit tests, but we should check the backend behavior.

Comment (binary-hamedanchi) [inline src/LiveApi.js]: I think it's better to not pass an extra parameter when it's not needed or doesn't have value.

Comment (aminmarashi) [inline src/LiveApi.js]: The question is does the backend code handle `&brand=` like when it's not passed?

Comment (raunakkathuria) [inline src/LiveApi.js]: why we need to empty parameter? its not a mandatory parameter so we should only add it as and when needed. Backend will be able to handle it if you pass it blank thats not a problem.

Comment (aminmarashi) [inline src/LiveApi.js]: Then better to change: 
```
const urlPlusParams = `${this.apiUrl}?l=${this.language}&app_id=${this.appId}&brand=${this.brand}`; 
```

Comment (binary-hamedanchi) [inline src/LiveApi.js]: I changed that part to make it look simpler.

File: src/LiveApi.js [hunk 2]
```diff
@@ -81,7 +83,8 @@ export default class LiveApi {
     }
 
     connect(connection: ?WebSocket): void {
-        const urlPlusParams = `${this.apiUrl}?l=${this.language}&app_id=${this.appId}`;
+        const optionalParam = this.brand ? `&brand=${this.brand}` : '';
```
Comment (aminmarashi) [inline src/LiveApi.js]: More general:
```
const optionalParams = this.brand ? `brand=${this.brand}` : '';
const urlPlusParams = `${this.apiUrl}?l=${this.language}&app_id=${this.appId}&${optionalParams.join('&')}`;
```

**Pull request merged**